jenkins/artifacts/JenkinWorkflow

```diff
@@ -0,0 +1,96 @@
+pipeline {
+    agent {label 'buildserver'}
+
+    environment {
+        targetParentLocation = "/opt/home/nightly/"
+        harvestPath = "harvest"
+        imageName = "rahulguptajss/harvest"
+        VERSION=""
+        RELEASE=""
+    }
+
+    stages {
+
+        stage('clean workspace') {
+            steps {
+                cleanWs()
+                sh 'rm -rf $targetParentLocation$VERSION$RELEASE'
+
+            }
+        }
+        stage('Setup GO') {
+            steps {
+                sh '''
+                wget -q -O go.tar.gz "https://dl.google.com/go/$(curl https://golang.org/VERSION?m=text).linux-amd64.tar.gz"
+                rm -rf /usr/local/go && tar -C /usr/local -xzf go.tar.gz
+                '''
+            }
+        }
+
+        stage('Download Prerequisites') {
+            steps {
+                sh '''
+               apt-get update
+               apt-get install -y rpm
+               apt-get install -y net-tools
+               apt install -y git-all
+               apt-get install -y build-essential
+                '''
+            }
+        }
+
+        stage('Git Clone Harvest') {
+            steps {
+               sh '''
+                #git clone --single-branch --branch $GIT_BRANCH https://github.com/NetApp/harvest.git
+                git clone --single-branch --branch main https://github.com/NetApp/harvest.git
```

Comment:
may be rename the jenkins file if branch name is going to main only.

Reply:
for testing purpose i have kept main. will revert it to branch name